app/pages/home-not-logged-in/promoted.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import counterpart from 'counterpart';
+import { Link } from 'react-router';
+import ProjectCard from '../../partials/project-card';
+
+counterpart.registerTranslations('en', {
+  promotedHomePage: {
+    discover: 'What Will You Discover?',
+    participate: `
+      Participate in research of all kinds, from classifying galaxies to counting
+      penguins to transcribing manuscripts. Whatever your interest, there's a
+      Zooniverse project for you.
+    `
+  }
+});
+
+const HomePagePromoted = ({ promotedProjects }) => {
+  return (
+    <section className="home-promoted">
+
+      <div className="home-promoted__content">
+        <Translate className="tertiary-kicker" content="promotedHomePage.discover" />
+        <Translate className="display-body" content="promotedHomePage.participate" />
+      </div>
+
+      <div className="home-promoted__cards">
+        {promotedProjects.map((project) => {
+          return (
+            <ProjectCard
+              key={project.id}
+              customCSS="home-page-not-logged-in__project-card"
+              imageSrc={project.image}
+              landingPage={true}
+              project={project}
+            />);
+        })}
+      </div>
+
+      <Link to="/projects" className="primary-button primary-button--light">See All Projects</Link>
+    </section>
+  );
+};
+
+HomePagePromoted.propTypes = {
+  promotedProjects: React.PropTypes.arrayOf(React.PropTypes.object)
```

Comment:
I'm pretty sure the linter kicks off about using `object`, you can use `shape()` and pass in an object of required properties instead